### PR TITLE
Fix/Slop-45/Movie Submissions Not Marked as 'Draft'

### PR DESCRIPTION
**Pull request by dani9401**
## Describe your changes
This change sets the defaultValue of "status" on each new movie/slop created to "draft". I made a small edit to the movie model to include a defaultValue moving forward.

## Issue ticket number and link
https://tripleten-apiary.atlassian.net/jira/software/projects/SLOP/boards/2?selectedIssue=SLOP-45

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [ ] If there are changes to custom resolvers, I have added / updated automated tests
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by marzahlde09** [comment]
I agree with your changes to movie.ts and the schema!

The only ALTER in the migration file so far is for adding foreign key dependencies. Should you be using ALTER TABLE to add a default value?

I'd assume the migration file would want the default added to CREATE TABLE `Movie`

**Comment by dani9401** [comment]
> I agree with your changes to movie.ts and the schema!
> 
> The only ALTER in the migration file so far is for adding foreign key dependencies. Should you be using ALTER TABLE to add a default value?
> 
> I'd assume the migration file would want the default added to CREATE TABLE `Movie`

Hey Danny - thanks for the review! I'm actually not familiar with an Alter Table...but after you mentioned the term I will be doing some researching on that, so thanks for the tip!
I'm going to bring this up in Discord so I can start to understand this a bit more :)

**Comment by marzahlde09** [comment]
I think we could all use a rundown on some of the unfamiliar backend
topics. The migration file is new to me - will that update the live server
if we make changes? I don’t know exactly how that works either!

If you want to learn more about some database terminology I can help with
that (I can interpret probably 95% of what you’re seeing on the backend)
just let me know and we can schedule a zoom or something.

-Danny

**Review by tomershvueli — CHANGES_REQUESTED** [review]
@dani9401 taking a look at this now, and I'm realizing you're working off of an old version of this repo. If you go to your branch you'll see '5 commits behind main': 
image

To fix this, in your branch you can do the following (assuming you have no unstaged changes):
`git checkout main`
`git pull origin main`
`git checkout <your branch name`
`git merge main`
Here you might get some merge requests, depending on which changes you're missing. 
Once you're up to date, you'll only really need 1 migration file. What was the reason for 3? One migration file can hold multiple SQL statements necessary for a single migration, so we should really only try to have 1 new migration file per PR unless there's a good reason otherwise.

**Comment by dani9401** [comment]
@tomershvueli Thank you for catching this! Resubmitting now.

**Comment by dani9401** [comment]
Team - this is now ready for 2 Peer Reviews when someone gets a chance!

Per Tomer's notes above, unneeded migration files have been removed and the branch is no longer behind main :)

**Review by ATribbey — APPROVED** [review]
As per the story requirements, created slops are appearing as drafts in the DB, and the drafted slop properly appears in the 'View Submitted' page. 

Additionally, looking through your changes helped me understand a bit more the things that go into new migrations and editing the existing DB schema. 

Well done! 

PR-45

**Review by fdlai — APPROVED** [review]
Hi! I only just realized this was updated and in need of another review. Yep, everything seems to work as specified in the AC. Good work!

I did notice a bug though that can be a new task:

After initially creating a new slop for the 1st time with that draft status, we can't see the **View submitted** slops link, until we refresh the page.

File: migrations/20240426200342_new_movies_created_are_set_to_draft_status_by_default/migration.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE `movie` MODIFY `status` VARCHAR(191) NULL DEFAULT 'draft';
```
Comment (tomershvueli) [inline migrations/20240426200342_new_movies_created_are_set_to_draft_status_by_default/migration.sql]: When I try to run `npm run dev` on this branch it warns me it's about to delete a table `movie`, whereas our table name is `Movie` - note the capital M. Let's fix this to make sure prisma knows we're talking about the right table.

**Review by tomershvueli — CHANGES_REQUESTED** [review]
We need to ensure capitalization of tables are consistent here. Also let's run the linter here so we can try and get a green build.

File: migrations/20240426200342_new_movies_created_are_set_to_draft_status_by_default/migration.sql (hunk 1 above)
Comment (dani9401) [reply migrations/20240426200342_new_movies_created_are_set_to_draft_status_by_default/migration.sql]: @tomershvueli Thank you!! I'm definitely learning here :) I have updated and pushed with changes.

**Review by tomershvueli — APPROVED** [review]
This works great! Nicely done!

**Pull request merged**